chainimport/headers_import_test.go

```diff
@@ -193,6 +194,546 @@ func TestTargetStoreFreshnessDetection(t *testing.T) {
 	}
 }
 
+// TestHeaderMetadataRetrieval tests the header metadata retrieval from the
+// import sources. It checks that the header metadata is retrieved correctly
+// from the import sources.
+func TestHeaderMetadataRetrieval(t *testing.T) {
+	t.Parallel()
+	type Prep struct {
```

Comment:
Same here re the internal/exporter functions. 

I like this prep/verify pattern generally.